api/spec/v1.yaml

```diff
@@ -1903,6 +1903,11 @@ components:
             Retrieving the other signers is currently not supported by this API.
           # NOTE: To expose the other signers, use the transaction_signers table.
           example: oasis1qz670t637yyxshnlxhjj5074wgwl94d0x5x69zqd
+        sender_eth_0:
+          type: string
+          description: |
+            The Ethereum address of this transaction's 0th signer.
+          example: 0xd8A2Ae03f6Edd58999a0F1005db7a6532F2AA79e
```

Comment:
I think our output is all-lowercase. Also, do we prefix the `0x`?

Reply:
> I think our output is all-lowercase.

Yes, it is!

> Also, do we prefix the 0x?

Not that I know of.

I thought that mixed case would be more consistent with [explorer](https://testnet.explorer.sapphire.oasis.dev/tx/0x837d852282ba29546f04f57d535a38879d9eca530b22b37e52ed9c95349c82c1) and [Etherscan](https://etherscan.io/address/0x1f9090aae28b8a3dceadf281b0f12828e676c326)? I don't feel strongly about this and can go either way.

Reply:
The mixed case is an eth kinda-cool kinda-hacky feature where the lower/uppercase distinction forms a series of bits, and those bits represent the checksum of the address. It's optional; all eth tools should accept all-lowercase addresses. We don't implement the mixed-case anywhere (we'd have to recompute it) right now.

Reply:
`ethCommon.HexToAddress` which we use does return the mixed-case.

Reply:
Oh my bad, TIL. Thank you.